clients/web/src/views/layout/LoginView.js

```diff
@@ -54,10 +54,6 @@ var LoginView = View.extend({
             }, this));
         },
 
-        'click a.g-register-link': function () {
```

Comment:
I don't think you want to be removing this, since you're not updating the template for *this* particular view.